src/Geometry/Geometry.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Geometry.swift
+//  Created by Logan Wright
+//
+
+import Foundation
+import UIKit
+
+public extension Int {
+    public var radians: CGFloat {
+        return CGFloat(self).radians
+    }
+}
+public extension CGFloat {
+    public var radians: CGFloat {
+        return CGFloat(Double(self) * (M_PI / 180))
+    }
+}
+
+public extension CGFloat {
+    public var degrees: CGFloat {
+        return self * CGFloat(180.0 / M_PI)
+    }
+}
```

Comment:
Because I made them at different times, and threw them all in this file :)  Will fix.